### PR TITLE
Fix pytest

### PR DESCRIPTION
**Pull request by semihalf-bis-maciej**
This is "local" pull request before up-stream

This patch fixes expect* methods from utils.py as well as call of this methods
Test need to raise exception to failure can be catch by pytest
Also '(' is special char for regex so need to be escaped

@semihalf-adamczyk-konrad @semihalf-wojtas-marcin please review and be forgiving

### CODE REVIEW

**Comment by adamczyk-konrad** [comment]
This patch shall rather not be a part of this MR: https://github.com/Plasteli-net/svunit/pull/1/commits/cf8eef243f41126099535b3c1774c05c42768c36.

**Comment by adamczyk-konrad** [comment]
In both https://github.com/Plasteli-net/svunit/pull/1/commits/44422216f6d465770311fc385fa2543d0c204f3e  and https://github.com/Plasteli-net/svunit/pull/1/commits/3f7769db49b602599b93b9a2d7550d06764353d9 commit messages, there're missing dots at the end of sentences.

File: test/utils.py [hunk 1]
```diff
@@ -127,15 +127,21 @@ def verify_testrunner(testrunner, ts0, ts1='', ts2='', ts3='', tr=''):
 
 def expect_testrunner_pass(logfile_path):
     with open(logfile_path) as file, mmap.mmap(file.fileno(), 0, access=mmap.ACCESS_READ) as log:
-        return re.search(br'INFO:  \[.*\]\[testrunner\]: PASSED (. of . suites passing) \[$SVUnitVersion\]', log)
+        if re.search(br'INFO:  \[.*\]\[testrunner\]: PASSED \(. of . suites passing\) \[.*\]', log):
```
Comment (adamczyk-konrad) [inline test/utils.py]: I'm maybe missing something, but why you've replaced `$SVUnitVersion` with `.*` regex syntax here?

Commit message does not say anything about it. Maybe there was another other bug that you've fixed here too?

Please elaborate more about this change.

Comment (semihalf-bis-maciej) [reply test/utils.py]: Yes, this is bug fix. In log message we have version number of SVUnit in this place e.g.: "SVUnit v3.34.0" but test check if there is "$SVUnitVersio" string not proper value of version. In test env we don't have access to version number so I use ".*" instead
But yes, maybe this should be elaborate in commit log
I will add this, thx

Comment (adamczyk-konrad) [reply test/utils.py]: Good enough for me, +1.

**Comment by adamczyk-konrad** [comment]
+1

**Pull request merged**